CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+**_Contribution Control Checklist_**
+
+
+
+
+
+
+
+**Sections**
+
+1- Starting Point
+
+2- Integration
+
+3- Levelling up
```

Comment:
```suggestion
3- Levelling up
```

Reply:
So here is my first real test huh, Let's do solve it

Reply:
I try to achieve something, but not going the way i want it to go haha. But it will eventually.

Reply:
Ok so i did manage to work a little, but i had hard time to find myself . I should be good to go for further work now. I need to go right now but i be back working later  tonight.... 

I wanted to do some fantasy and impress you, but well... I will do some later you can be 100 % sure about it